CoreFindView/src/au/gov/asd/tac/constellation/views/find2/components/BasicFindTab.java

```diff
@@ -140,19 +143,29 @@ public BasicFindTab(final FindViewTabs parentComponent) {
         // inAttributesMenu
         inAttributesMenu.setOnContextMenuRequested(event -> contextMenu.show(inAttributesMenu, event.getScreenX(), event.getScreenY()));
 
-        // set the action for changing the seleciton in the
-        // currentSelctionChoiceBox
-        currentSelectionChoiceBox.getSelectionModel().selectedItemProperty().addListener(new ChangeListener<String>() {
+        // set the action for changing the seleciton in the postSearchChoiceBox
+        searchInChoiceBox.getSelectionModel().selectedItemProperty().addListener(new ChangeListener<String>() {
             @Override
             public void changed(ObservableValue<? extends String> observableValue, String oldElement, String newElement) {
                 updateSelectionFactors();
+                updateBasicFindParamters();
+            }
+        });
+
+        // set the action for changing the seleciton in the postSearchChoiceBox
+        postSearchChoiceBox.getSelectionModel().selectedItemProperty().addListener(new ChangeListener<String>() {
```

Comment:
can the listener here and the one above be converted to a lambda? Or is there an issue with doing that? (If there's a problem, make the parameters final in the function)